### PR TITLE
Update test/e2e/scheduling/nvidia-gpus to also run CUDA10.1 vector add.

### PR DESCRIPTION
**Pull request by chardch**
**What type of PR is
/kind cleanup

**What this PR does / why we need it**:
This is to test the latest CUDA version, 10.1

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/scheduling/nvidia-gpus.go [hunk 1]
```diff
@@ -71,6 +71,16 @@ func makeCudaAdditionDevicePluginTestPod() *v1.Pod {
...
+				// CUDA 10.1
+				{
+					Name:  "vector-addition-cuda10-1",
+					Image: imageutils.GetE2EImage(imageutils.CudaVectorAdd3),
+					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							gpuResourceName: *resource.NewQuantity(1, resource.DecimalSI),
+						},
+					},
+				},
```
Comment (jiayingz) [inline test/e2e/scheduling/nvidia-gpus.go]: I think most of gpu tests are set up with 2 gpus on each machine, so running with three containers each requesting a gpu would make the gpu job not schedulable. I guess we perhaps can replace cuda8 container with cuda10.1 now.

Comment (chardch) [reply test/e2e/scheduling/nvidia-gpus.go]: Would we want to increase the number of gpus to 3 for the tests in order to make sure one older version still works, or is testing 10 and 10.1 sufficient?

**Pull request closed without merging**